packages/front-end/components/UpgradeMessage.tsx

```diff
@@ -43,10 +43,24 @@ export default function UpgradeMessage({
 
   // Self-hosted
   return (
-    <div className="alert alert-info">
-      Purchase a commercial license key to {upgradeMessage}. Contact{" "}
-      <a href="mailto:sales@growthbook.io">sales@growthbook.io</a> for more
-      info.
-    </div>
+    <a
+      className="boxlink cta mb-3"
+      href="#"
+      onClick={(e) => {
+        e.preventDefault();
+        showUpgradeModal();
+      }}
+    >
+      <div className="alert alert-info m-0">
+        <p className="mb-0">
+          Upgrade to a commercial license key to {upgradeMessage}.
+        </p>
+        {accountPlan === "oss" && (
```

Comment:
Use `!isCloud()`

Reply:
this component should actually be deleted in favor of `components/Marketing/UpgradeMessage.tsx`, which does switch on `isCloud()`